### PR TITLE
Add WebAssembly support

### PR DESCRIPTION
**Pull request by wanderer**
this add support for detecting webassembly binarys

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -102,6 +102,10 @@
     "xo": "*"
   },
   "xo": {
-    "esnext": true
+    "esnext": true,
+    "rules": {
```
Comment (SamVerschueren) [inline package.json]: Do not turn these rules off, I will fix them in master later today.

Comment (SamVerschueren) [inline package.json]: Fixed them in master. So now you can rebase upon master and it should work.

**Comment by sindresorhus** [comment]
You also need to add a link in the readme and package.json keyword.

Can you link to the source for the "magic number" used here?

**Comment by wanderer** [comment]
ok @sindresorhus documentation on the magic number and readme updated!

**Comment by sindresorhus** [comment]
I meant here, not in the source code, but I can fix that. For reference:

```js
// The Magic number is  0x6d736100 (i.e., '\0asm') see https://github.com/WebAssembly/design/blob/master/BinaryEncoding.md#high-level-structure
```

**Comment by sindresorhus** [comment]
Thank you :)

**Pull request merged**